### DISCUSSION TITLE
使用Markdown书写文章时, 如何使用自定义的CSS样式

### DISCUSSION THREAD

**Discussion by 762610973** [hugo-fixit/FixIt#448] [Q&A] [answered]
使用Markdown书写文章时, 如何使用自定义的CSS样式
例如: `<span style="color: red">span tag color test</span>`
我发现并不能正确渲染出**红色**, 翻了官网的文档, 没找到😥

**Comment by Lruihao** [comment, marked as answer]
有两种方式：

1. 直接写 HTML，需要设置 `goldmark.renderer.unsafe` 为 `true`, 不然 HTML 不会被渲染。

    ```md
    <span style="color: red">span tag color test</span>
    ```

2. 使用自定义属性，Hugo 支持向 Markdown 块添加属性（例如 CSS 类），例如 表格、列表、段落等，需要设置 `goldmark.parser.attribute.block` 为 `true`。更多详情请看[文档](https://fixit.lruihao.cn/zh-cn/documentation/content-management/markdown-syntax/extended/#%e8%87%aa%e5%ae%9a%e4%b9%89%e5%b1%9e%e6%80%a7)。

    ```md
    span tag color test
    {.text-danger}
    ```

**Reply by 762610973** [reply]
非常感谢您😁, 我找到了unsafe的设置, 也非常抱歉, 没有翻阅更多的帖子直接来问您😭
